modules/distr-tracing-tempo-object-storage-setup-aws-sts-install.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-aws-sts-install_{context}"]
+= Installing the AWS Security Token Service (STS)
+
+???
+
+.Procedure
+
+. Create AWS S3 bucket.
+
+. Create a custom AWS IAM Role associated in a trust relationship with the `ServiceAccount` of the TempoStack instance:
++
+[source,yaml]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Principal": {
+          "Federated": "arn:aws:iam::${<aws_account_id>}:oidc-provider/${<oidc_provider>}" # <1>
+        },
+        "Action": "sts:AssumeRoleWithWebIdentity",
+        "Condition": {
+          "StringEquals": {
+            "${OIDC_PROVIDER}:sub": [
```

Comment:
@pavolloffay, what I don't understand is whether the AWS is used only for storage or for running Red Hat OpenShift Service on AWS (ROSA) in this use case?